ec2/launch_template/templates/scripts/user_data.sh

```
@@ -0,0 +1,161 @@
+else
+    log "Mackerel agent configuration not found in Parameter Store, using default"
+fi
+
+log "Mackerel Agent configuration completed"
+%{ endif }
+
+# ==================================================
+# カスタムユーザーデータの実行
+# ==================================================
+
+%{ if custom_user_data != "" }
```

Shellcheck (0.10.0)

[warning] This is interpreted as a command name ending with '{'. Double check syntax.

(SC2288)